infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/InitializedState.java

```diff
@@ -33,12 +33,25 @@ public InitializedState(LogReplicationFSM logReplicationFSM) {
     public LogReplicationState processEvent(LogReplicationEvent event) throws IllegalTransitionException {
         switch (event.getType()) {
             case SNAPSHOT_SYNC_REQUEST:
-            case SNAPSHOT_WAIT_COMPLETE:
                 // Set the id of the event that caused the transition to the new state
                 // This is used to correlate trim or error events that derive from this state
                 LogReplicationState snapshotSyncState = fsm.getStates().get(LogReplicationStateType.IN_SNAPSHOT_SYNC);
                 snapshotSyncState.setTransitionEventId(event.getEventID());
                 return snapshotSyncState;
+
+            case SNAPSHOT_WAIT_COMPLETE:
```

Comment:
Can we please update the LogReplicationFSM diagram to reflect this new event?

Reply:
The complete flow is not there yet. Will change the FSM accordingly for the next PR.